editor/src/components/custom-code/code-file.ts

```diff
@@ -151,7 +151,7 @@ export function componentDescriptor(
 
 export interface ComponentDescriptorWithName extends ComponentDescriptor {
   componentName: string
-  moduleName: string
+  filenameNoExtension: string
```

Comment:
is this the file path with folders? or just the filename? won't it clash if there's two files with the same name? I searched the codebase and nothing seems to _set_ componentName or filenameNoExtension

Reply:
FYI we discussed this, and agreed on moduleName and a comment about it